spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -916,6 +930,53 @@ public void setReadCompacted(boolean readCompacted) {
 			this.readCompacted = readCompacted;
 		}
 
+		public Startup getStartup() {
+			return this.startup;
+		}
+
+	}
+
+	public static class Startup {
+
+		/**
+		 * The max time to wait for the container to start.
+		 */
+		private Duration timeout;
```

Comment:
Is there a default for this property? If so I think we should use the opportunity to set the default so that it's documented, and very in a test that it's consistent in case it changes.